test/cli/rbi/test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+main/sorbet --silence-dev-message --censor-for-snapshot-tests \
+  --rbi $'module Foo; sig { void }; def self.foo; end; end' \
+  -e $'Foo.foo(123)' 2>&1
```

Comment:
This test doesn't actually test that the string passed to `--rbi` is treated as an RBI.

Maybe you want to try to add an overloaded sig here?

Reply:
Good idea, I've added a better test.